manager/integration/tests/test_node.py

```diff
@@ -350,6 +350,27 @@ def test_replica_scheduler_no_disks(client):  # NOQA
     common.wait_for_volume_delete(client, vol_name)
 
 
+@pytest.mark.node  # NOQA
+def test_replica_zone_anti_affinity(client):  # NOQA
+    """
+    Test replica scheduler with zone anti-affinity
+
+    1. Set zone anti-affinity to hard, and replica count to 2.
+    2. Create 2 nodes and add the same zone label to them.
```

Comment:
- we automatically deploy Longhorn on 3 node cluster for integration tests, so may be we can do like this:
  1- Label nodes 1 & 2 with  same zone label  e.g `zone1` 
  2- Label nodes 3 with another zone label  e.g `zone2` 
  3- Disabled scheduling on node 3

Reply:
Thanks for the input. I will update this part, but for "Disabled scheduling on node 3", we don't need to change the scheduling part of the node, we only need to change the node's zone label.